lib/ddtrace/configuration/settings.rb

```diff
@@ -65,16 +92,38 @@ def initialize(*_)
           end
         end
 
+        # Internal tracer {Datadog::Statsd} metrics collection.
+        #
+        # The list of metrics collected can be found in {Datadog::Ext::Diagnostics::Health::Metrics}.
         settings :health_metrics do
```

Comment:
Minor: Should we not mention the tracer here, since this is in a generic group and other products may end up using it as well?